keyboards/ergodox_ez/keymaps/dvorak_42_key/keymap.c

```diff
@@ -447,44 +457,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
```

Comment:
Would you be able to completely get rid of this function?

Reply:
yes I can, this was the recommended approach for macros some years back, but I can convert those to the modern way.

Reply:
Yep, I'm trying to remove `fn_actions` and `action_get_macro` as things like direct keycodes (eg. `MO()`) and `process_record_user()` with custom keycodes have basically superseded them.

Reply:
I've removed `action_get_macro` function